client/components/OrderStage/OrderStage.jsx

```diff
@@ -3,23 +3,50 @@ import React from 'react';
 import './OrderStage.scss';
 
 function OrderStage(props) {
-  const { title, time, orders, id } = props;
+  const { title, orders, id, onDelete } = props;
+  const time = Math.max(...orders.map((order) => order.time));
+  const diff = Math.floor((Date.now() - time) / 1000);
+
+  const intervals = [
+    [3600, ' hours'],
+    [60, ' minutes'],
+    [10, '0 seconds'],
+  ];
+  const stringParams = intervals.reduce(
+    (accumulator, currentValue) =>
+      currentValue[0] <= accumulator[0] && accumulator[1] === 'just now'
+        ? currentValue
+        : accumulator,
+    [diff, 'just now']
+  );
+
+  const timeString =
+    stringParams[1] === 'just now'
+      ? stringParams[1]
+      : `${Math.floor(diff / stringParams[0])}${stringParams[1]} ago`;
 
   const counts = {};
 
   orders.forEach((elem) => {
     counts[elem.item._id] = (counts[elem.item._id] || 0) + 1;
   });
 
+  const getCallbackById = (orderId) => () => onDelete(orderId);
+
   const orderList = Object.keys(counts).map((key) => {
-    const item = orders.find((elem) => elem.item._id === key)?.item;
+    const { item, _id } = orders.find((elem) => elem.item._id === key);
     return (
       <div className="order">
         <figure>
           <img src={item?.imgPath} alt={item?.title} />
           <span>{item?.title}</span>
         </figure>
-        <div className="count">{counts[key]}</div>
+        <span>
+          <div className="count">{counts[key]}</div>
+          <button type="button" onClick={getCallbackById(_id)}>
+            X
```

Comment:
@IgorChaikin Could you please also implement this counter component, which allows to change amount of items in order to `+1` or `-1`

<img width="75" alt="image" src="https://user-images.githubusercontent.com/12561302/129893003-a19279e0-d0b1-46e5-ac8a-a0f28b500c08.png">

`X` button should remove all items from cart (3 pizzas) for example